OpenNefia.Content/Prototypes/BuffPrototype.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OpenNefia.Core.Prototypes;
+using OpenNefia.Core.Serialization.Manager.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.Prototypes
+{
+    public enum BuffType
+    {
+        Blessing,
+        Hex,
+        Food
+    }
+
+    [Prototype("Elona.Buff")]
+    public class BuffPrototype : IPrototype
+    {
+        [DataField("id", required: true)]
+        public string ID { get; } = default!;
+
+        [DataField]
```

Comment:
Could document what asset this is meant to be used with:

```csharp
///<summary>
/// Asset region ID for use with <see cref="Protos.Asset.BlessingHexIcons"/>.
///</summary>
```

Reply:
changed in 2e3ad22